Yoti.Auth.Sandbox/Profile/Request/ExtraData/ThirdParty/SandboxIssuingAttributes.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Yoti.Auth.Sandbox.Profile.Request.ExtraData.ThirdParty
+{
+    public class SandboxIssuingAttributes
+    {
+        [JsonProperty(PropertyName = "expiry_date")]
+        public DateTime? ExpiryDate { get; private set; }
```

Comment:
Is there any way we can test that this date is always formatted as RFC3339 with milliseconds?

Reply:
I can see this was addressed in 68256c3